tutorials/luigi-app-react/luigi-app-react.md

```diff
@@ -191,24 +189,48 @@ In this step, you will create another HTML file which will serve as a template f
 
 In this step, we configure `webpack` and adjust dependencies in order to make it easier to develop and build the app.
 
-1. Go to `react-core-mf/config/webpack.config.js`
+1. Go to `react-core-mf/webpack.config.js`
 
-2. Find the following line from the file and [comment it out](https://www.w3schools.com/js/js_comments.asp) by surrounding it with `/*` or `//` tags:
+2. **Note** the following line. By default our `webpack-dev-server` will redirect all requests to `index.html` which is our app's main entry point where the Luigi Core library is injected.
```

Comment:
Don't use the label "Note" and use the correct formatting for notes as described in the Tutorials style guide.
Also, there is a comma missing feore "which".